ansible_collections/arista/avd/examples/single-dc-l3ls/README.md

```diff
@@ -595,55 +665,168 @@ As an example, here is the configuration for `dc1-leaf1-server1`:
+examples/single-dc-l3ls/deploy-cvp.yml
+--8<--
+```
+
+1. At the highest level, the name and scope of the playbook is set, which in this example is the CloudVision server named `CLOUDVISION`.
+2. This task uses the role `arista.avd.eos_config_deploy_cvp` that pushes the generated configuration to the devices in scope.
+3. Specifies to use v3 of the collection.
+
+### Testing AVD output without a lab
+
+It is possible to use the `build.yml` playbook without any actual devices. The playbook will generate all of the output (variables, configurations, documentation), but will not attempt to communicate with any devices.
 
 Please look through the folders and files described above to learn more about the output generated by AVD.
 
-### Executing the playbook
+### Playbook Run
+
+To build the configurations files, run the playbook called `build.yml`.
+
+``` bash
+### Build Configurations and Documentation
+ansible-playbook playbooks/build.yml
+```
+
+After the playbook run finishes, EOS CLI intended configuration files were written to `intended/configs`.
+
+To build and deploy the configurations to your switches directly, using eAPI, run the playbook called `deploy.yml`. This assumes that your Ansible host has access and authentication rights to the switches. Those auth variables are defined in FABRIC.yml.
+
+``` bash
+### Deploy Configurations to Devices using eAPI
+ansible-playbook playbooks/deploy.yml
+```
+
+To build and deploy the configurations to your switches using CloudVision Portal, run the playbook called `deploy-cvp.yml`. This assumes that your CloudVision Portal server has access and authentication rights to the switches. Those auth variables are defined in FABRIC.yml.
+
+``` bash
+### Deploy Configurations to Devices Using CloudVision Portal
+ansible-playbook playbooks/deploy-cvp.yml
+```
+
+### EOS Intended Configurations
+
+Your configuration files should be similar to these.
+
+=== "dc1-spine1"
+
+    ``` shell
+    --8<--
+    examples/single-dc-l3ls/intended/configs/dc1-spine1.cfg
+    --8<--
+    ```
+
+=== "dc1-spine2"
+
+    ``` shell
+    --8<--
+    examples/single-dc-l3ls/intended/configs/dc1-spine2.cfg
+    --8<--
+    ```
+
+=== "dc1-leaf1a"
+
+    ``` shell
+    --8<--
+    examples/single-dc-l3ls/intended/configs/dc1-leaf1a.cfg
+    --8<--
+    ```
+
+=== "dc1-leaf1b"
+
+    ``` shell
+    --8<--
+    examples/single-dc-l3ls/intended/configs/dc1-leaf1b.cfg
+    --8<--
+    ```
+
+=== "dc1-leaf1c"
+
+    ``` shell
+    --8<--
+    examples/single-dc-l3ls/intended/configs/dc1-leaf1c.cfg
+    --8<--
+    ```
+
+=== "dc1-leaf2a"
+
+    ``` shell
+    --8<--
+    examples/single-dc-l3ls/intended/configs/dc1-leaf2a.cfg
+    --8<--
+    ```
+
+=== "dc1-leaf2b"
+
+    ``` shell
+    --8<--
+    examples/single-dc-l3ls/intended/configs/dc1-leaf2b.cfg
+    --8<--
+    ```
+
+=== "dc1-leaf2c"
+
+    ``` shell
+    --8<--
+    examples/single-dc-l3ls/intended/configs/dc1-leaf2c.cfg
+    --8<--
+    ```
+
+### Executing the playbooks
```

Comment:
This seems redundant to the section above `### Playbook Run`

Reply:
Removed headline